server/hl7/hl7.controller.js

```diff
@@ -70,6 +70,9 @@ function parseFile(req, res, next) {
   }).catch(err =>
     next(new APIError(err, httpStatus.BAD_REQUEST))
   ).then((data) => {
+    if (!data.startsWith('MSH|')) {
```

Comment:
This approach works since (I believe) all HL7 messages have to start with the message header (MSH)